modules/virt-importing-vm-cli.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+[id='virt-importing-vm-cli_{context}']
+= Importing a Red Hat Virtualization virtual machine with the CLI
+
+You can import a Red Hat Virtualization virtual machine with the CLI by creating the Secret and VirtualMachineImport Custom Resources (CRs). You apply the VirtualMachineImport configuration to start the import process and you can follow its progress.
+
+.Procedure
+
+. Create the Secret CR to store the RHV Manager credentials:
++
+[source,yaml]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rhv_credentials
+  namespace: default <1>
+type: Opaque
+stringData:
+  ovirt: |
+    apiUrl: "https://<RHVM_FQDN>:8443/ovirt-engine/api" <2>
+    username: admin@internal
+    password: <3>
+    caCert: |
+      -----BEGIN CERTIFICATE-----
+      <4>
+      -----END CERTIFICATE-----
+----
+<1> Optional. You can specify a different namespace.
+<2> Specify the FQDN of the RHV Manager.
+<3> Specify the password for `admin@internal`.
+<4> Specify the link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.3/html-single/rest_api_guide/index#obtaining_the_ca_certificate[RHV Manager CA certificate].
+
+. Create the VirtualMachineImport CR to configure the virtual machine import process:
++
+[source,yaml]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v2v.kubevirt.io/v1alpha1
+kind: VirtualMachineImport
+metadata:
+  name: vm_import
+  namespace: default
+spec:
+  providerCredentialsSecret:
+    name: rhv_credentials
+    namespace: default
+  targetVmName: vm_example
+  startVm: true
+  source:
+    ovirt:
+      vm:
+        id: <80554327-0569-496b-bdeb-fcbbf52b827b> <1>
+      mappings:
+        networkMappings:
+          - source:
+              name: <rhv_logical_network>/<vnic_profile> <2>
+            target: <target_network> <3>
```

Comment:
Right after target: there should not be anything. <target_network> <3. here is wrong.

Reply:
<3> is a tag - see draft on https://importing_rhv_cnv--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/importing_vms/virt-importing-rhv-vm.html#virt-importing-vm-cli_virt-importing-rhv-vm

![Selection_999(936)](https://user-images.githubusercontent.com/316242/87006547-87411c80-c1c9-11ea-959f-e23ca6df7173.png)

Reply:
Yes, it's a "callout" to explain that this value needs to be filled in.